### PR TITLE
Use ngettext properly in OverlayBar

### PR DESCRIPTION
**Pull request by jeremypw**

### CODE REVIEW

File: src/View/Widgets/OverlayBar.vala [hunk 1]
```diff
@@ -208,19 +208,19 @@ namespace Marlin.View {
...
                     if (files_count > 0) {
-                        str += ngettext (_(" and %u other item (%s) selected").printf (files_count, fsize),
-                                         _(" and %u other items (%s) selected").printf (files_count, fsize),
-                                         files_count);
+                        str += ngettext (" and %u other item (%s) selected",
+                                         " and %u other items (%s) selected",
+                                         files_count).printf (files_count, fsize);
                     } else {
                         str += _(" selected");
                     }
```
Comment (ryonakano) [inline src/View/Widgets/OverlayBar.vala:216]: Honestly I think we may want to avoid this way of constructing a translatable string but would be better to address in another PR.

**Review by ryonakano — APPROVED** [review]
Looks good to me.

**Comment by jeremypw** [comment]
Yes, this was just a minimal PR.  There is a possibility OverlayBar will be deprecated in future anyway.
